share/templates.d/99-generic/runtime-cleanup.tmpl

```diff
@@ -267,16 +234,10 @@ removefrom linux-firmware /usr/lib/firmware/as102*
 removefrom linux-firmware /usr/lib/firmware/qcom/venus*/*
 removefrom linux-firmware /usr/lib/firmware/meson/vdec/*
 removefrom lldpad /etc/*
-removefrom lua /usr/bin/*
-removefrom madan-fonts /usr/share/fonts/madan/*
 removefrom mdadm /etc/*
 removefrom mesa-dri-drivers /usr/${libdir}/dri/*_video.so
-removefrom module-init-tools /etc/* /usr/sbin/insmod.static /usr/sbin/weak-modules
-removefrom mt-st /etc/* /usr/sbin/*
+removefrom mt-st /usr/sbin/*
 removefrom mtools /etc/*
-removefrom ncurses /usr/bin/captoinfo /usr/bin/infocmp /usr/bin/infotocap
```

Comment:
this part breaks s390x/ppc64le image creation, because they install `ncurses` rpm, so it leaves the tools, but their libs are removed

Reply:
@AdamWill We could either restore this removal (I'm not sure why this line was added actually) or we could wrap the `removefrom ncurses-libs /usr/${libdir}/libpanel.* /usr/${libdir}/libtic*` line in an arch check for ppc64le and s390x

Reply:
I'm trying to figure out why s390x and ppc64le add `ncurses` to the package set, it could be a bug somewhere, hopefully will know more on Monday

Reply:
so the dep chain is following - powerpc-utils/s390utils -> perl stuff -> `perl-Term-Cap` -> `/usr/bin/infocmp` -> `ncurses`

`infocmp` call in `perl-Term-Cap` is a fall-back path and even this path has another fall-back, so my suggestion is to remove all of `ncurses` in the cleanup template with
`removefrom ncurses /usr/bin/*`

unrelated to this issues is a dependency on `ncurses` in the `@Core` group in `comps`, I believe this should be `ncurses-base` instead (the terminal definitions are needed, not the compiler/decompiler tools)

Reply:
Thanks for tracking that down. May as well remove all of the ncurses files, see PR #1085